NOMENCLATURE.md

```diff
@@ -72,46 +72,47 @@ maintains a connection to the _dispatcher_, waiting for the current set of
 tasks assigned to the node. Assigned tasks are then dispatched to the engine.
 The agent notifies the _dispatcher_ of the current state of assigned tasks.
 
-This is roughly analagous to an entertainment agent. It ensures the worker has
-the correct set of work and let's others know what the worker is doing.
+This is roughly analagous to an entertainment (What is this?) agent. It ensures
```

Comment:
Ah yes! Thanks for looking this over. I put that in there to discuss what is an entertainment agent. Are we talking about and actual person that represents an individual who is in the entertainment business, or is it some sort of design pattern, or something else?